benchmarks.md

```diff
@@ -3,89 +3,117 @@ layout: base
 
-
-The Arbiter will provide built-in support for the following benchmark options.
+The Arbiter provides built-in support for the following benchmark
+options. Additional implementation details are available in the
+``solarforecastarbiter-core``
+[documentation](https://solarforecastarbiter-core.readthedocs.io/en/latest/reference-forecasts.html).
 
 ### Intrahour horizons
 
-
 * Persistence
 * Persistence of clear sky index
-* ARIMA
 
+The Arbiter provides reference implementations of persistence
+algorithms. The averaging window of the Arbiter's benchmark persistence
+forecasts are fixed to be equal to a forecast's run length, but no
+longer than 1 hour. For example, a benchmark forecast for a 15 minute
+interval will use the most recent 15 minutes of data to compute the
+quantity to persist (irradiance, power, or clear sky index). Persistence
+of clear sky index accounts for the average clear sky index during an
+interval rather than simply using the interval start or end time.
 
 ### Intraday and longer horizons
 
-
-* HRRR irradiance
-* RAP irradiance
+* HRRR subhourly irradiance
+* RAP cloud cover to irradiance
 * NAM cloud cover to irradiance
 * GFS cloud cover to irradiance
 
-The Arbiter will derive irradiance forecasts from NAM and GFS cloud cover
-forecasts to properly account for the solar position and time averaging. The
-Arbiter will include options for bias-corrected NWP-based benchmark forecasts.
-
+With the exception of the HRRR, the Arbiter derives irradiance and power
+forecasts from NWP cloud cover forecasts to accurately account for solar
+position, inverval labels (beginning or ending), and interval averaging.
+For these models, the process is:
+
+1. Load hourly (or longer) interval data from the NWP grib files.
+   * For GFS cloud cover, unmix the mixed-intervals average data.
+2. Resample data to 5 minute intervals.
+   * For GFS cloud cover, backfill the data.
+   * For all other NWP data, interpolate the data.
+3. Convert cloud cover to irradiance following Larson et. al. (2016).
+4. If desired, use site metadata to compute AC power using
+   [pvlib-python](https://pvlib-python.readthedocs.io/en/latest/)
+   functions.
+5. Compute hourly averages with desired interval labels.
 
 ### Probabilistic
 
-
-* HRRRE
-
-Calibration of probabilistic benchmark forecasts is a topic of discussion.
-
+* GEFS cloud cover to ranked irradiance ensemble
+* Climatology (not yet implemented)
+* Persistence ensemble (not yet implemented)
+
+### Automated generation
+
+Operational benchmark forecasts are currently produced for every
+reference forecast site in the Arbiter's database. Forecasts are currently only produced
+using the [intraday and longer horizons](#Intraday-and-longer-horizons)
+models described above. For these automated benchmark forecasts, we choose to define
+[forecast attributes](/definitions/#forecastattrs) such that new forecasts
+would be issued at midnight local time for each site. Daylight Saving Time
```

Comment:
@alorenzo175 can you confirm this? 

Also would like to get some persistence forecasts going, but can update this when that happens.

Reply:
Not quite. The issue time is defined in the local time for each site, but it isn't midnight for all of forecasts, only GFS.

 The table is correct for GFS and NAM. HRRR and RAP actually run every 6 hours so that there is a continuous series.

Reply:
accurate now?

Reply:
yes, although if you don't know the jargon, you might not understand that the rap and hrrr are issued mulitple times. Maybe something like ``0Z, 6Z, 12Z, 18Z`` for the issue time would make it clear enough?